config.example.toml

```diff
@@ -25,6 +25,21 @@ master_key = "feffe9928665731c6d6a8f9467308308feffe9928665731c6d6a8f9467308308"
 tenant_public_key = ""  # The tenant public key to which the communication will be established with
 locker_private_key = "" # the locker private key to used used and the private key present with the tenant
 
-[kms]
+# To protect secret/sensitive values like:
+# - database.passwod
+# - secrets.master_key
+# - secrets.tenant_public_key
+# - secrets.locker_private_key
+# 
+# Following possible encryption schemes are used, of of them are mutually exclusive, the sections are:
+# - aws_kms (AWS KMS Symmetric Encryption)
+# - vault_kv2 (HashiCorp Vault Secrets Engine KV version 2)
+
+
+[aws_kms]
```

Comment:
Same comment as the feature flag.  We can name these similar to the feature flag names with `kms-` or `kms_` being the shared common prefix.  Kind of indicates that these are all alternatives.